<launch>
    <!-- ======雷达标定参数====== -->
    <!-- 标定点1在基坐标系下的位置 -->
    <param name="marker1_x" value="8.2296" type="double" />
    <param name="marker1_y" value="0" type="double" />
    <!-- 标定点2在基坐标系下的位置 -->
    <param name="marker2_x" value="30.1752" type="double" />
    <param name="marker2_y" value="0" type="double" />
    <!-- 雷达在基坐标系下的位置 -->
    <param name="lidar_x" value="23.1752" type="double" />
    <param name="lidar_y" value="-2.5" type="double" />

    <!-- ======通讯参数====== -->
    <!-- 主机ip地址 -->
    <param name="master_ip_address" value="192.168.101.3" type="str" />
    <!-- 主机发送指令的端口号 -->
    <param name="master_command_port" value="1145" type="int" />
    <!-- 主机表达“开始发送数据”的命令 -->
    <param name="master_pose_send_command" value="SR" type="str" />
    <!-- 主机表达“开始录制”的命令 -->
    <param name="master_begin_record_command" value="LS" type="str" />
    <!-- 主机表达“结束录制”的命令 -->
    <param name="master_end_record_command" value="LE" type="str" />

    <!-- ======检测参数====== -->
    <!-- 激光雷达参数 -->
    <param name="lidar_frame_id" value="rslidar" type="str" />
    <param name="lidar_pc_topic" value="/rslidar_points" type="str" />
    <param name="vertical_num" value="32" type="int" />
    <param name="horizon_num" value="1800" type="int" />
    <param name="vertical_accuracy" value="1" type="double" />
    <param name="horizon_accuracy" value="0.2" type="double" />
    <param name="min_angle" value="-16" type="double" />
    <param name="max_angle" value="15" type="double" />


    <node pkg="curling_detect" type="udp_client.py" name="Lidar_Client" output="screen" />

    <node pkg="curling_detect" type="prompt_graph" name="prompt_graph" output="screen" />
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find curling_detect)/config/prompt_graph.rviz" required="true"/>

</launch>